src/js/components/Chart/README.md

```diff
@@ -204,6 +220,7 @@ full
     large
     xlarge
     full
+    auto
```

Comment:
Why aren't we allowing `auto` for `height` as well as for `width`?

Reply:
There's nothing to automatically set the height from. Width comes from the number of values + thickness + gap.